### ISSUE TITLE
Guide to running on windows.

### ISSUE THREAD

**Issue by LAP87** [rivenmedia/riven#617]
Install Git, Node.js and Python 3 first if you haven't already.
Git: https://git-scm.com/downloads
Node.js https://nodejs.org/en/download/prebuilt-installer/current
Python 3.12+ https://www.python.org/downloads/

Reboot.

open CMD/Powershell in C: drive twice.

Shell -1
```
git clone https://github.com/rivenmedia/riven.git && cd riven
pip install poetry
poetry install --without dev
poetry run python src/main.py
```

This will launch the backend, you can minimize this window now and continue with the frontend.

Shell -2
```
git clone https://github.com/rivenmedia/riven-frontend.git && cd riven-frontend
npm install
npm run preview
```
This launches the frontend and spits out a localhost:XXXX adress where you can access riven from any browser.
Minimize this window also. (If you accidentally leftclick anywhere on the prompts it will pause the program, press ESC once to resume)
Someone might be able to figure out how to easily make these run hidden from boot, but that is not me.

**Comment by dreulavelle** [comment]
the issue isnt getting the app to run on windows, its plex that doesnt like the symlinks. Maybe JF and Emby would work with the windows setup

**Comment by LAP87** [comment]
> the issue isnt getting the app to run on windows, its plex that doesnt like the symlinks. Maybe JF and Emby would work with the windows setup

Yeah, it stopped running after completing the setup and crashes when i try to restart it.. dang!
[riven-20240814-0133.log](https://github.com/user-attachments/files/16606443/riven-20240814-0133.log)

**Comment by dreulavelle** [comment]
those crashes look likes Riven cant find the database. Make sure postgres is installed

**Comment by LAP87** [comment]
> those crashes look likes Riven cant find the database. Make sure postgres is installed

Installed the full suite PostgresQL for win 64 and rebooted, now there's a different error, maybe you understand it better? 
[riven-20240814-1729.log](https://github.com/user-attachments/files/16615507/riven-20240814-1729.log)
